Tasks/NuGetInstaller/task.json

```diff
@@ -1,4 +1,4 @@
-{
+﻿{
```

Comment:
This looks like a BOM change. BOM changes will break the build, so make sure you check in exactly what was checked out. (Sorry for vagueness here - I can never remember whether we're supposed to have BOMs or not.)